js/evidence.js

```diff
@@ -1,26 +1,196 @@
-(function ($, window) {
+(function ($, window, document, evidence) {
 
-	//'use strict';
+	'use strict';
+
+	var $filters = $('#story-filters'),
```

Comment:
We haven't talked about this at all... :grin: ... but I'd like to start pushing for using jQuery's JS style guide - https://contribute.jquery.org/style-guide/js/. I wouldn't necessarily worry about changing all of this code, as it's a project in progress, but we should talk about it for the future. The biggest adjustment is the use of `"` rather than `'` and more liberal spacing, similar to WordPress and PHP.